examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/provisioning/DeviceProvisioningFragment.kt

```diff
@@ -132,7 +139,30 @@ class DeviceProvisioningFragment : Fragment() {
       if (thread != null) {
         network = NetworkCredentials.forThread(NetworkCredentials.ThreadCredentials(thread.operationalDataset))
       }
-
+      deviceController.setDeviceAttestationFailureCallback(600
```

Comment:
600 is a magic number, any reason you chose this?

Can we put this into a `static final int` or something and maybe add a comment maybe clarifying how we arrived at this integer?